serializers/src/main/java/io/pravega/schemaregistry/serializers/BaseSerializer.java

```diff
@@ -0,0 +1,21 @@
+/**
+ * Copyright (c) Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.schemaregistry.serializers;
+
+import io.pravega.client.stream.Serializer;
+
+import java.nio.ByteBuffer;
+
+abstract class BaseSerializer<T> implements Serializer<T> {
+    @Override
+    public final T deserialize(ByteBuffer serializedValue) {
+        throw new IllegalStateException();
```

Comment:
I'm still seeing these `IllegalStateException` s. Did you say you were changing them to unsupported operation? If so, then perhaps search for all occurrences and change them all.

Reply:
You are perhaps looking at the outdated file diff because i have changed it in both BaseSerializer.java and BaseDeserializer.java.